plugins/src/main/kotlin/extension/locales.kt

```diff
@@ -9,7 +9,7 @@ val locales = setOf(
     "de",
     "el",
     "en",
-    "en_US",
+    "en-rUS",
```

Comment:
Just to double check, did you run `importSupportedLocalesFromLocalazy.py` to update this?